uint8 ACTION_AVAILABLE=0
uint8 ACTION_DELETED=1
uint8 ACTION_USING=2
uint8 ACTION_RELEASE=3
uint8 ACTION_PROCESSING=4

uint8 PROCESS_VAD=0
uint8 PROCESS_ASR=1
uint8 PROCESS_ID=2

uint8 action
string path
uint16 node_pid

AudioBatchMetadata metadata
